web3/_utils/module_testing/eth_module.py

```diff
@@ -42,13 +40,23 @@
 from web3._utils.method_formatters import (
     to_hex_if_integer,
 )
+from web3._utils.module_testing.utils import (
```

Comment:
another nit: I think I like `web3._utils.module_testing_utils.py` a little bit better, but am open to having it in the module testing folder if you think that makes more sense.

Reply:
I like that better too 👌🏼